src/api/MarkerBasedAPI.js

```diff
@@ -49,6 +50,7 @@ class MarkerBasedApi extends Base {
         limit: number,
         shouldFetchAll: boolean,
         requestData: Object = {},
+        headers?: StringMap,
```

Comment:
Where is this being used?

Reply:
Removed! It'll be in history if it needs to be added back